rasa/utils/common.py

```diff
@@ -309,7 +309,10 @@ def run_in_loop(
     result = loop.run_until_complete(f)
 
     # Let's also finish all running tasks:
-    pending = asyncio.Task.all_tasks()
+    try:
+        pending = asyncio.all_tasks(loop)
+    except AttributeError:
+        pending = asyncio.Task.all_tasks()
```

Comment:
This is a fallback for python 3.6, right? Would be great to add a comment about it

Reply:
Right! I'll add that asap